spec/spec_helper.rb

```diff
@@ -1,5 +1,5 @@
-$:.unshift File.expand_path("../../lib", __FILE__)
```

Comment:
I'm pretty sure @phiggins is going to tell me we don't need these

Reply:
You're right, I don't think you need either of them.